Source/ACE.Server/Entity/Fellowship.cs

```diff
@@ -174,6 +174,9 @@ public void AddConfirmedMember(Player inviter, Player player, bool response)
             }
 
             UpdateAllMembers();
+
+            if (inviter.CurrentMotionState.Stance == MotionStance.NonCombat) // only do this motion if inviter is at peace, other times motion is skipped. 
```

Comment:
the standard way to do this would be inviter.CombatMode == CombatMode.NonCombat, but this should still be fine